patterns/page-cv-bio.php

```diff
@@ -24,10 +24,10 @@
 			<div class="wp-block-column" style="flex-basis:65%">
 				<!-- wp:group {"layout":{"type":"flex","orientation":"vertical","justifyContent":"stretch"}} -->
 				<div class="wp-block-group">
-					<!-- wp:paragraph {"align":"left","style":{"typography":{"fontSize":"22rem","letterSpacing":"-0.03em"}}} -->
-					<p class="has-text-align-left" style="font-size:22rem;letter-spacing:-0.03em"><?php echo esc_html_x( 'Hey,', 'Example heading above the content of the CV/Bio pattern.', 'twentytwentyfive' ); ?></p>
-					<!-- /wp:paragraph -->
-
+					<!-- wp:heading {"textAlign":"left","className":"wp-block-heading","style":{"typography":{"fontSize":"22rem","letterSpacing":"-0.03em","fontStyle":"normal","fontWeight":"300","lineHeight":"1.4"}}} -->
```

Comment:
This custom className is not needed. It will show up in the additional CSS field in the editor.

Reply:
Fix in recent commit